src/pages/Login.jsx

```diff
@@ -1,10 +1,10 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import { Card, CardBody, Input, Link } from "@nextui-org/react";
 import InputForm from "../components/InputForm";
-import { useState } from "react";
 import PasswordEye from "../components/PasswordEye";
-import { login } from "../utils/authHandler";
-import { useNavigate } from "react-router-dom";
 import LoadingButton from "../components/LoadingButton";
+import { login } from "../utils/authHandler";
```

Comment:
(praise): I love that you're grouping relevant imports together. It really helps with readability!